km_api/know_me/tests/serializers/test_emergency_item_serializer.py

```diff
@@ -51,13 +58,20 @@ def test_serialize(
         context=serializer_context)
 
     url_request = api_rf.get(item.get_absolute_url())
+    serializer_request = serializer_context['request']
+    has_read = media_resource.has_object_read_permission(serializer_request)
```

Comment:
Created the has_read and has_read variables in some places because of line length constraints. Would it be better to use this format everywhere?